### PR TITLE
[vrotsc-types] (#347) Add remaining Aria plugin types definitions

### PR DESCRIPTION
**Pull request by unbreakabl3**
### Checklist

- [ ] I have added relevant error handling and logging messages to help troubleshooting
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation, relevant usage information (if applicable)
- [x] I have updated the PR title with affected component, related issue number and a short summary of the changes introduced
- [x] I have added labels for implementation kind (kind/*) and version type (version/*)
- [ ] I have tested against live environment, if applicable
- [ ] I have synced any structure and/or content vRA-NG improvements with vra-ng and ts-vra-ng archetypes (if applicable)
- [ ] I have my changes rebased and squashed to the minimal number of relevant commits. **Notice: don't squash all commits**
- [x] I have added a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### Related issues and PRs
https://github.com/vmware/build-tools-for-vmware-aria/issues/347

### CODE REVIEW

**Comment by unbreakabl3** [comment]
Hi @VenelinBakalov ,
this is a last one. All classes were added.

File: vro-types/o11n-plugin-aria/index.d.ts [hunk 1]
```diff
@@ -2186,3 +2186,678 @@ declare class VraMachineBootConfig {
...
+	 */
+	detachMachineDisk(machineId: string, diskId: string): VraRequestTracker;
+
+	/**
+	 * Attach machine disk to a machine.
+	 * @param attachMachineDisk
+	 * @param machineId
+	 */
+	attachMachineDisk(
+		attachMachineDisk: VraDiskAttachmentSpecification,
```
Comment (VenelinBakalov) [inline vro-types/o11n-plugin-aria/index.d.ts:2326]:
```suggestion
		attachmentSpecification: VraDiskAttachmentSpecification,
```

File: vro-types/o11n-plugin-aria/index.d.ts [hunk 2]
```diff
@@ -2186,3 +2186,678 @@ declare class VraMachineBootConfig {
...
+	 * @param genericRestClient
+	 */
+	setGenericRestClient(genericRestClient: VraGenericRestClient): void;
+}
+
+/**
+ * HostManager provides all the CRUD operations for VMware Aria Automation Plugin along with Generic Rest Client support.
+ */
+declare class VraHostManager {
+	readonly defaultHostData;
```
Comment (VenelinBakalov) [inline vro-types/o11n-plugin-aria/index.d.ts:2786]:
```suggestion
	readonly defaultHostData: VraHost;
```

File: vro-types/o11n-plugin-aria/index.d.ts [hunk 3]
```diff
@@ -2186,3 +2186,678 @@ declare class VraMachineBootConfig {
...
+	 * @param props
+	 */
+	save(props: Object): string;
+
+	/**
+	 * Update Automation Host object.
+	 * @param props
+	 * @param host
+	 */
+	update(props: Object, host: VraHost): string;
```
Comment (VenelinBakalov) [inline vro-types/o11n-plugin-aria/index.d.ts:2799]:
```suggestion
	update(props: Object, host: VraHost): void;
```

File: vro-types/o11n-plugin-aria/index.d.ts [hunk 4]
```diff
@@ -2186,3 +2186,678 @@ declare class VraMachineBootConfig {
...
+
+	/**
+	 * Creates a dynamic Automation Host.
+	 * @param properties
+	 */
+	createHost(properties: Object): VraHost;
+
+	/**
+	 * Creates a temporary host with the current user host and credentials.
+	 * @param id
```
Comment (VenelinBakalov) [inline vro-types/o11n-plugin-aria/index.d.ts:2809]:
```suggestion

```

**Comment by VenelinBakalov** [comment]
> Hi @VenelinBakalov , this is a last one. All classes were added.

That's awesome! thank you so much for extending all of these, it is quite big effort!

**Review by VenelinBakalov — APPROVED** [review]

**Pull request merged**